Engines/Wine/Engine/Versions/script.js

```diff
@@ -122,15 +122,16 @@ module.getAvailableVersions = function (wizard) {
 
 
 module.getLatestStableVersion = function (wizard, architecture) {
-    return getLatestVersion(wizard, "upstream-linux-" + architecture, /^\d+\.0(\.\d+)?$/);
+    return getLatestVersion(wizard, `${this._wineDistribution}-${this._winePackage}-${architecture}`, /^\d+\.0(\.\d+)?$/);
```

Comment:
The idea was more:
```Javascript
module.getLatestStableVersion = function (wizard, distribution, package, architecture) {
    return getLatestVersion(wizard, `${distribution}-${package}-${architecture}`, /^\d+\.0(\.\d+)?$/);
```
Honestly, I don't even understand how the current implementation can work, e.g. `this._wineDistribution` should not exist here.

Reply:
I added the field wineDistribution to QuickScript.